extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -157,8 +157,9 @@ class LifecycleConductor {
                     next);
             },
         ], err => {
-            if (err && err.Throttling) {
-                log.info('not starting new lifecycle batch', { reason: err });
+            // check for err.is until errors from bucketClient have the new Arsenal format.
+            if (err && err.is && err.is.Throttling) {
```

Comment:
If we're not sure which error version we have, the following is probably safer

```
if (err?.Throttling || err?.is?.Throttling)
```
Otherwise you're losing the error type check if the error is in the old version.

Reply:
Take note that TS will whine a lot `Throttling does not exists on err`, just put a `@ts-ignore` the previous line and you'll be good to go.

Reply:
This isn't TS yet :wink: